src/app/todo/todo-list/todo-list.component.ts

```diff
@@ -17,13 +16,15 @@ import {  addTodoReq,  delTodoReq, getTodosReq, updateTodoReq } from 'src/app/st
 export class TodoListComponent implements OnInit{
   
   todos$: Observable<ReadonlyArray<Todo>>;
-  
+  isLoading$: Observable<boolean>;
+
   constructor(
     private logger: NGXLogger,
     private store: Store
   ) { 
-    this.todos$=this.store.select(selectTodos)
+    this.todos$ = this.store.select(selectTodos)
     this.logger.info(this.todos$)
+    this.isLoading$ = this.store.select(selectLoading);
```

Comment:
not a major thing. just a suggestion in the future.

we would like to move line 27 above line 26. And use `.debug` if you really want to print out the object.